example/datasets/image_net/dataset.py

```diff
@@ -0,0 +1,49 @@
+import xmltodict
+
+from starwhale import Link, Image, BoundingBox  # noqa: F401
+
+
+def to_box_view(bndbox):
+    xmin = int(bndbox["xmin"])
+    ymin = int(bndbox["ymin"])
+    xmax = int(bndbox["xmax"])
+    ymax = int(bndbox["ymax"])
+    return BoundingBox(xmin, ymin, xmax - xmin, ymax - ymin)
+
+
+def do_iter_item():
+    import requests
+
+    response = requests.get(
+        "https://starwhale-examples.oss-cn-beijing.aliyuncs.com/dataset/image-net/ILSVRC/ImageSets/CLS-LOC/val.txt"
+    )
+    for line in response.text.splitlines():
+        img_name = line.split()[0]
+        anno_link = f"https://starwhale-examples.oss-cn-beijing.aliyuncs.com/dataset/image-net/ILSVRC/Annotations/CLS-LOC/val/{img_name}.xml"
+        while True:
```

Comment:
`while True` in this code block do the retry task?  and it may be an infinite retry?

Reply:
https://github.com/star-whale/starwhale/blob/main/client/starwhale/utils/retry.py is the starwhale http retry .

Reply:
done